internal/dag/secret.go

```diff
@@ -27,6 +27,15 @@ import (
 // CACertificateKey is the key name for accessing TLS CA certificate bundles in Kubernetes Secrets.
 const CACertificateKey = "ca.crt"
 
+// validTLSSecret returns true if the Secret contains certificate and private key material.
```

Comment:
```suggestion
// validTLSSecret returns true if the Secret is of type TLS and contains certificate and private key material.
```

Reply:
@skriss I've applied the changes, hope they're good